lib/client-assets.php

```diff
@@ -402,7 +410,7 @@ function gutenberg_register_scripts_and_styles() {
 	wp_register_style(
 		'wp-editor',
 		gutenberg_url( 'build/editor/style.css' ),
-		array( 'wp-components', 'wp-editor-font' ),
+		array( 'wp-components', 'wp-editor-font', 'wp-nux' ),
```

Comment:
Should we do the same for 'wp-edit-post'?

Reply:
👌 0499f5c82